build-system/pr-check/build-targets.js

```diff
@@ -112,9 +145,18 @@ const targetMatchers = [
       );
     },
   },
+  {
+    targets: ['OWNERS'],
+    func: file => {
+      return isOWnersFile(file);
+    },
```

Comment:
Any reason not to just do `func: isOwnersFile,` ?

Reply:
This is moot. See latest commit.